.gitignore

```diff
@@ -181,3 +181,5 @@ Temporary Items
 .apdisk
 
 *.before
+
+.atom-build.json
```

Comment:
It's a configuration file for an Atom package _(I've now switched to Github's Atom editor because it's infinitely better than Dream Maker)_ that basically tells it what to compile and with what compiler, instead of compiling from Dream Maker.
http://i.imgur.com/WtmmwFO.png
Github is making me either commit the file itself or add it in the .gitignore and commit that.